tests/unit/test_session.py

```diff
@@ -781,9 +781,7 @@ def test_internal_component(self):
             self.session._get_internal_component('internal'), component
         )
         with self.assertRaises(ValueError):
-            # get_component has been deprecated to the public
-            with pytest.warns(DeprecationWarning):
-                self.session.get_component('internal')
+            self.session.get_component('internal')
```

Comment:
For reference, we're handing the correct case [here](https://github.com/boto/botocore/pull/3109/files#diff-7824b4fcea790555e99c27ab732bde7187f87896ab367892c6ec1b8949c064f4R801-R806).